xml/compute.xml

```diff
@@ -497,39 +497,39 @@
    <!-- href="https://fate.novell.com/324153" -->
    <title>HYPRE &mdash; scalable linear solvers and multigrid methods</title>
    <para>
-    HYPRE is a library of linear solvers which aim to solve large
-    and detailed simulations faster than traditional methods at
-    large scales.
+    HYPRE is a library of linear solvers which aim to solve large and detailed
+    simulations faster than traditional methods at large scales.
    </para>
    <para>
-    The library offers a comprehensive suite of
-    scalable solvers for large-scale scientific simulation,
-    featuring parallel multigrid methods for both structured
-    and unstructured grid problems. HYPRE is highly portable
-    and supports a number of languages, and is developed at
-    Lawrence Livermore National Laboratory.
+    The library offers a comprehensive suite of scalable solvers for large-scale
+    scientific simulation, featuring parallel multigrid methods for both
+    structured and unstructured grid problems. HYPRE is highly portable, and
+    supports a number of languages, and is developed at Lawrence Livermore
```

Comment:
I think the subtext is "don't ask us how it works. We're simply computer jockeys. Ask those guys."

Reply:
I don't think that subtext was consciously added then and it would in fact run counter to our usual argument that people pay us because that's how they get support for or software that is otherwise gratis.

Reply:
OK. Fair point. Lends it extra gravitas?

Reply:
... Or we just copied a description from somewhere off the internet: https://gitlab.com/arm-hpc/packages/-/wikis/packages/hypre

In any case, I am setting this resolved, this is imo not worth discussing more. :)